src/arith/domain_touched.cc

```diff
@@ -34,18 +35,48 @@ namespace arith {
 
 using namespace tir;
 
+struct LoadAccess {
```

Comment:
Can we use `using LoadAccess = std::vector<std::vector<IntSet>> set;`?

Reply:
I'm currently using the struct type as sugar for compile-time lookup with std::get to index into the std::tuple. That said I took your advice an added an alias for the buffer touches. Happy to do it whichever way is preferable, I just found the tuple to be nice here.

Reply:
I don't have strong opinions about this. Both approaches are clear enough!